index.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+</head>
+<body>
+    <header>
+        <div class="nav-bar-logo">
+            <div class="logo">My Logo</div>
+            <a href="#" class="bars-mob"></a>
+        </div>
+        <div class="container-1">
+            <div class="discription">
+                <h1>I'm Girma Glad to see you!</h1>
+                <p>I'm a software developer! I can help you build a product, feature or website look through some of my work and experience! if you like what you  see and have a project you need coded, don't hestiate to contact me</p>
+                <div class="let-s">
+                    <h1>LET'S CONNECT</h1>
+                    <div class="vector">
+                        <ul>
+                            <li><a href="https://github.com/girma336" class="twitter"></a></li>
+                            <li><a href="#" class="linkedin"></a></li>
+                            <li><a href="#" class="medium"></a></li>
+                            <li><a href="#" class="github"></a></li>
+                            <li><a href="#" class="hand"></a></li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </header>
+    <!-- Second task start from here -->
+    <div class="grid-container">
+        <section class="main-section">
+            <article>
+                <div class="main-1"> 
+                    <div class="image-1"></div>
+                    <div class="tonic-dic">
+                        <h1>Tonic</h1>
+                        <ul>
+                            <p>CANOPY</p>
+                            <li>Back End developer</li>
+                            <li>2015</li>
+                        </ul>
+                    </div>
+                    <div class="text">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </div>
+                    <div class="nav-source">
+                        <ul>
+                            <li>html</li>
+                            <li>JavaScript</li>
+                            <li>css</li>
+                        </ul>
+                    </div>
+                    <div class="button-m">
+                        <button>See Project</button>
```

Comment:
[OPTIONAL] Please note that your project's buttons don't match the figma design. Its good pracice to have your buttons match figma to develop your css skills better

- [ ] Your design

<img width="350" alt="Screenshot 2022-09-26 at 13 59 28" src="https://user-images.githubusercontent.com/48546992/192260582-a618a94f-9b05-4a12-8946-2525a4c5acb1.png">

Figma template
<img width="169" alt="Screenshot 2022-09-26 at 14 02 33" src="https://user-images.githubusercontent.com/48546992/192260769-f51bb131-8110-4399-a16d-ed71b06a8131.png">